spec/models/user_spec.rb

```diff
@@ -12,57 +12,69 @@
   end
 
   it "is able to access personal group" do
-    p_group = user.create_personal_group
+    p_group = user.personal_group
     expect(user.exists_personal_group?).to be(true)
     expect(user.personal_group).to eq(p_group)
   end
 
   it "ensures personal group exists to access it" do
-    expect(user.exists_personal_group?).to be(false)
-    expect { user.personal_group }.to raise_error(StandardError)
+    user2 = FactoryBot.build(:user)
+    expect(user2.exists_personal_group?).to be(false)
+    expect { user2.personal_group }.to raise_error(StandardError)
   end
 
   it "cannot create multiple personal groups" do
-    user.create_personal_group
     expect { user.create_personal_group }.to raise_error(StandardError)
   end
 
-  it "throws error if there are multiple personal groups" do
-    user.create_personal_group
+  it "cannot have multiple personal groups" do
     p_group = Group.create(name: "personal_group", tag: :personal_group)
     p_group_membership = Membership.create(group_id: p_group.id, user_id: user.id, role: :member)
     user.memberships.push(p_group_membership)
-    user.save
-    expect { user.exists_personal_group? }.to raise_error(StandardError)
-    expect { user.personal_group }.to raise_error(StandardError)
+    expect { user.save! }.to raise_error(ActiveRecord::RecordInvalid)
   end
 
-  it "is destroyed when user is destroyed" do
-    p_group = user.create_personal_group
+  it "personal group is destroyed when user is destroyed" do
```

Comment:
The tests should read as a sentence. i.e. `it "destroys the personal group when the user is destroyed"`